packages/builder/test/helpers/environment.ts

```diff
@@ -1,18 +1,25 @@
 import { resetBuilderContext } from "../../src/internal/reset";
-import { AlgobRuntimeEnv } from "../../src/types";
+import { AlgobRuntimeEnv, PromiseAny } from "../../src/types";
 
 declare module "mocha" {
   interface Context {
     env: AlgobRuntimeEnv;
   }
 }
 
-export function useEnvironment() {
-  beforeEach("Load environment", function () {
+export function useEnvironmentWithBeforeEach(beforeEachFn?: (algobRuntimeEnv: AlgobRuntimeEnv) => PromiseAny) {
+  beforeEach("Load environment", async function () {
     this.env = require("../../src/internal/lib/lib");
+    if (beforeEachFn) {
+      await beforeEachFn(this.env)
+    }
   });
 
   afterEach("reset builder context", function () {
     resetBuilderContext();
   });
 }
+
+export function useEnvironment() {
+  useEnvironmentWithBeforeEach()
+}
```

Comment:
Is this function needed?

Reply:
It may be removed but I wanted to document what I do with the complex one.
`useEnvironment` is used a lot.

Reply:
`useEnvironment` and `useEnvironmentWithBeforeEach` is the same thing. let's use one. If you prefer to use `useEnvironment`, then let's use that name and rename useEnvironmentWithBeforeEach to useEnvironment.